src/libraries/Common/src/Interop/Windows/Kernel32/Interop.CreateSymbolicLink.cs

```diff
@@ -55,17 +55,10 @@ internal static void CreateSymbolicLink(string symlinkFileName, string targetFil
 
```

Comment:
While you are on it, could you please also fix redundant invocations of `Environment.OSVersion.Version` above?
```
Version osVersion = Environment.OSVersion.Version;
bool isAtLeastWin10Build14972 = osVersion.Major >= 11 || osVersion.Major == 10 && osVersion.Build >= 14972;
```

Reply:
Done.